app/src/main/res/values/strings.xml

```diff
@@ -884,29 +884,29 @@
     <string name="network_log_app_name_unknown">Unknown</string>
     <string name="network_log_app_name_unnamed">Unnamed (%1$s)</string>
 
-    <string name="dns_crypt_connection_failure">Error connecting to DNSCrypt server.</string>
-    <string name="dns_proxy_connection_failure">Error connecting to DNS Proxy server.</string>
-    <string name="dns_proxy_connection_failure_lockdown">Excluding %1$s for %2$s in lockdown mode can cause Internet connectivity issues.</string>
+    <string name="dns_crypt_connection_failure">Could not connect to DNSCrypt server.</string>
+    <string name="dns_proxy_connection_failure">Could not connect to the server of the DNS proxy.</string>
+    <string name="dns_proxy_connection_failure_lockdown">Excluding %1$s for %2$s in lockdown can cause Internet connectivity issues.</string>
     <string name="dns_proxy_toast_parameter">DNS Proxy</string>
     <string name="socks5_proxy_toast_parameter">SOCKS5 Proxy</string>
     <string name="orbot_toast_parameter">Tor-as-a-proxy</string>
 
-    <string name="system_dns_connection_failure">Cannot determine the current DNS server</string>
+    <string name="system_dns_connection_failure">Cannot determine current DNS server</string>
 
     <string name="copy_clipboard_label">URL</string>
 
     <string name="update_complete_snack_message">New version downloaded.</string>
-    <string name="update_complete_action_snack">RESTART</string>
+    <string name="update_complete_action_snack">Restart</string>
 
-    <string name="settings_local_blocklist_in_use">%1$s blocklists in-use</string>
+    <string name="settings_local_blocklist_in_use">%1$s blocklists in use</string>
     <string name="settings_local_blocklist_dialog_positive">Download</string>
     <string name="settings_local_blocklist_dialog_negative">Cancel</string>
     <string name="settings_local_blocklist_version">Version: v%1$s</string>
 
     <string name="accessibility_notification_title">Action Required</string>
-    <string name="accessibility_notification_content">RethinkDNS accessibility service has stopped or crashed. No longer blocking backgrounded apps. Tap to re-enable.</string>
+    <string name="accessibility_notification_content">Rethink accessibility service stopped or crashed. No longer blocking backgrounded apps. Tap to turn it on again.</string>
 
-    <string name="accessibility_failure_toast">RethinkDNS accessibility service has stopped or crashed. No longer blocking backgrounded apps.</string>
+    <string name="accessibility_failure_toast">Rethink accessibility service stopped or crashed. No longer blocking backgrounded apps.</string>
```

Comment:
```suggestion
    <string name="accessibility_failure_toast">The accessibility service stopped or crashed. No longer blocking backgrounded apps.</string>
```

Reply:
This is incorrect: Rethink's accessibility service stopped. Not Android's.